docs/getting_started.md

```diff
@@ -12,10 +12,15 @@ dependencies {
 }
 ```
 
+Add external storage permission in your `AndroidManifest.xml` file
+```
+<uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
+```
```

Comment:
Thanks for the suggestion, however this is not required. If the `WRITE_EXTERNAL_STORAGE` permission is in the manifest then LeakCanary will show a notification asking for it. If it's not, LeakCanary will write to the app storage. This mean we minimize the impact LeakCanary has on the app. If we wanted to add the permission we could do that from the library, but we won't. I'd like to keep the getting started steps as simple as possible.

That being said, it might be valuable to add something to the recipes or FAQ to explain what's up with external storage and leakcanary.